components/supervisor-api/java/src/main/java/io/gitpod/supervisor/api/Control.java

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2023 Gitpod GmbH. All rights reserved.
```

Comment:
We should remove those changes: Copyright years don't have to be updated in existing files ([source](https://news.ycombinator.com/item?id=34880663)).

Also, is this an automatic change? If yes, we should track it down and fix it maybe :thinking:

Reply:
Yeah, it's been generated by `protoc`. I wish it didn't make so many changes myself too